sema4ai/vscode-client/src/robo/agentPackage.ts

```diff
@@ -253,9 +253,8 @@ export const updateAgentVersion = async (agentPath: string): Promise<void> => {
         const errorMsg = `Failed to update the agent at: ${agentPath}`;
         logError(errorMsg, error, "ERR_UPDATE_AGENT_VERSION");
 
-        const detailedErrorMsg = `${errorMsg}. Please check the 'OUTPUT > Sema4.ai' for more details.`;
-        OUTPUT_CHANNEL.appendLine(detailedErrorMsg);
-        window.showErrorMessage(detailedErrorMsg);
+        OUTPUT_CHANNEL.appendLine(errorMsg);
```

Comment:
If `logError(errorMsg)` is used, the `OUTPUT_CHANNEL.appendLine(errorMsg);` is not needed (as `logError` will already add that to the OUTPUT_CHANNEL).